src/pages/tickets/__tests__/Tickets.spec.tsx

```diff
@@ -255,6 +269,26 @@ describe('Tickets Component', () => {
     })();
   });
 
+  it('Test that org bounties are scrollable on org home page', async () => {
+    render(<Tickets />);
+
+    (async () => {
+      await waitFor(() => {
+        expect(screen.queryByTestId('tickets-component')).toBeInTheDocument();
+      });
```

Comment:
@okhot The Org Homepage FIle is OrgTickets, Tickets is the bounty homepage, so you are testing the bounty homepage here not the Org homepage, and there is a test for the bounty homepage scrolling already.